server/controllers/email-controller.js

```diff
@@ -25,7 +25,7 @@ exports.sendMail = async (email, subject, body) => {
     html: body, // html body
   });
 
-  console.log("Message sent: %s", info.messageId);
+  //console.log("Message sent: %s", info.messageId);
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.